server/game/cards/08.5-TFM/ContraintParLaFoi.js

```diff
@@ -0,0 +1,21 @@
+const _ = require('underscore');
+
+const PlotCard = require('../../plotcard.js');
+
+class ContraintParLaFoi extends PlotCard {
+    setupCardAbilities() {
+        this.whenRevealed({
+            handler: () => {
+                _.each(this.game.allCards.filter(card => card.power > 0), card => {
```

Comment:
It probably doesn't matter, as cards out of play shouldn't have any power on them, but you could also use ```this.game.filterCardsInPlay(card => card.power > 0)``` to iterate over, which is also slightly more descriptive.